docs/src/components/HomePageCard/HomePageCard.js

```diff
@@ -73,13 +73,13 @@ const HomePageCard = (props: Props) => {
         >
           <div
             className={getClassName('bpkdocs-home-page-card__image')}
-            style={{ backgroundImage: `url(${image})` }}
+            style={{ backgroundImage: `url(/${image})` }}
             alt={title}
           />
           {icon && (
             /* eslint-disable backpack/use-components */
             <img
-              src={icon}
+              src={`/${icon}`}
```

Comment:
But this doesn't happen on other pages?
I guess we should find out why the behaviour is inconsistent on the home page 🤔

Reply:
It's not as complex as it appears. The home page is fairly unique because it appears on `/` so we never knew this was a problem until now. Other pages run on other routes, so they'd have found this problem immediately during development

Reply:
Oh I thought other pages used `src: {screenshotPointerOffset}` but they actually do say `src: \`/${screenshotPointerOffset}\``